communications-handbook.md

```diff
@@ -0,0 +1,155 @@
+
+We strive for consensus in decisions regarding content policies, guidelines, and directions, as well as when questions about specific changes or new material arise. When consensus cannot be achieved within a reasonable timeframe, the issue should be referred upward within the organization. In addition, plans for large changes (such as new websites or communication channels) should be brought to the Steering Committee before they are implemented to ensure they are consistent with these guidelines and that various groups remain coordinated within the Association.
+
+This document lays out general policies for online content that have been encountered by the organization so far. It also provides guidance on what types of other changes warrant discussion among various committees before being implemented.
+
+# General Guidelines
+
+## Style Guide
+
+When referring to the organization:
+
+* US-RSE Association, US Research Software Engineer Association
+* US-RSE for short/abbreviation if desired after one of the above is used
+* US-RSE as an adjective/possessive: US-RSE website, US-RSE Steering Committee
+
+## Organizational account administration
+
+In order to maintain continuity of access, in cases where the organization manages group resources, such as the us-rse.org domain, the GitHub organization, the GSuite account, the Twitter account, and the Slack workspace, there should be at least two, and preferably three, Steering Committee members as administrators. Those administrators will be responsible for providing access to community members who are part of the relevant committee and workgroups responsible for the actual maintenance of the resources.
+
+When creating an account on behalf of the organization with a service or site, use the leadership@us-rse.org email address. Rules can be set up in this account if needed to forward the messages to others as needed. Contact the Steering Committee if access is needed to this (or for the time being, contact Christina Maimone directly, as she has the password and the two-factor device). The password for this account can be reset by the GSuite Admins if needed.
+
+
+## Use of the US-RSE Logo
+
+Official logo files can be found in the logo repository in the organizational GitHub: https://github.com/USRSE/logo. Logos from this repository should be used whenever a logo is needed. Variations in size or file format may be created from the source Adobe Illustrator files. Proposals for changes to the logo (colors, text, elements, etc.) or variants of the logo for specific uses should be submitted as a Pull Request. Agreement must be obtained from the Steering Committee before the PR is accepted or rejected. 
+
+Use of the logo implies organizational endorsement of whatever the logo is included on, even if that wasn't the intent.  Before the logo is used, please ensure there is consensus from a relevant committee (e.g. website, social) or the Steering Committee that use of the logo is appropriate. 
```

Comment:
I'm a little unsure about this. Does this mean I can't give a talk and use the logo on one of my slides to illustrate a statement I make about US-RSE?

Reply:
I understand the comment.  Other ideas how to handle this?  Do we have no restrictions until we have a logo that we want to trademark?  Or spell them out: 
```suggestion
Use of the logo implies organizational endorsement of whatever the logo is included on, even if that wasn't the intent.  Please do not use the logo to advertise or brand events or products that are not endorsed by the US-RSE?  When in doubt, please contact the Steering Committee.
```

Reply:
that seems good to me